app/src/main/res/values/colors.xml

```diff
@@ -11,12 +11,12 @@
   <color name="title_color">#ffffffff</color>
   <color name="sura_number_color">#7fffffff</color>
   <color name="sura_details_color">#b2ffffff</color>
-  <color name="transparent_actionbar_color">#AA000000</color>
+  <color name="transparent_actionbar_color">#aa000000</color>
   <color name="panel_background_color">#ff424242</color>
   <color name="translation_hdr_color">#ff777777</color>
   <color name="translation_sura_header">#fff1ecd8</color>
   <color name="translation_sura_header_night">#ff424242</color>
-  <color name="translation_verse_text_color">#ff858383</color>
+  <color name="translation_verse_text_color">#858383</color>
```

Comment:
Reminder to self to check why these changes were made to this file and revert if necessary.